etc/types/aws/actions/dir_exists.sh

```diff
@@ -1,7 +1,10 @@
 bucket=$1
-key=${2:1}
-region=$3
-not_exist=$($flight_SILO_types/aws/cli/bin/aws s3api head-object --bucket "$bucket" --key "$key" --no-sign-request --region "$region">/dev/null 2>&1; echo $?)
+test $2 = true && sign_request=--no-sign-request || sign_request=""
+key=${3:1}
+export AWS_DEFAULT_REGION=$4
+export AWS_ACCESS_KEY_ID=$5
+export AWS_SECRET_ACCESS_KEY=$6
```

Comment:
I'm not sure that these need to be `export`ed; as long as the values are being used in the same session, they should persist for as long as we need (and will be cleared once the script is over, as desired).

Regardless, if you use `open3` (see other comment), you will be able to access these variables by name, instead of by their ordered index.